src/test/java/seedu/contax/logic/commands/FindAppointmentCommandTest.java

```diff
@@ -0,0 +1,112 @@
+package seedu.contax.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.contax.commons.core.Messages.MESSAGE_APPOINTMENTS_LISTED_OVERVIEW;
+import static seedu.contax.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.contax.testutil.TypicalAppointments.APPOINTMENT_ALICE;
+import static seedu.contax.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.contax.commons.core.GuiListContentType;
+import seedu.contax.model.Model;
+import seedu.contax.model.ModelManager;
+import seedu.contax.model.Schedule;
+import seedu.contax.model.UserPrefs;
+import seedu.contax.model.appointment.Appointment;
+import seedu.contax.model.appointment.NameContainsKeywordsPredicate;
+import seedu.contax.model.appointment.PersonNameContainsKeywordsPredicate;
+import seedu.contax.testutil.AppointmentBuilder;
+
+
+public class FindAppointmentCommandTest {
+
+    @Test
+    public void equals() {
+        NameContainsKeywordsPredicate firstPredicate =
+                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
+        NameContainsKeywordsPredicate secondPredicate =
+                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+
+        FindAppointmentCommand findFirstCommand = new FindAppointmentCommand(firstPredicate);
+        FindAppointmentCommand findSecondCommand = new FindAppointmentCommand(secondPredicate);
+
+        // same object -> returns true
+        assertTrue(findFirstCommand.equals(findFirstCommand));
+
+        // same values -> returns true
+        FindAppointmentCommand findFirstCommandCopy = new FindAppointmentCommand(firstPredicate);
+        assertTrue(findFirstCommand.equals(findFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(findFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(findFirstCommand.equals(null));
+
+        // different person -> returns false
+        assertFalse(findFirstCommand.equals(findSecondCommand));
+    }
+
+    @Test
+    public void clientName_equals() {
```

Comment:
You should change the term client to person for consistency.

```suggestion
    public void personName_equals() {
```

Reply:
Fixed